insights/parsers/nmcli.py

```diff
@@ -191,3 +192,39 @@ def parse_content(self, content):
     def disconnected_connection(self):
         """(list): It will return all the disconnected static route connections."""
         return self._disconnected_connection
+
+
+add_filter(Specs.nmcli_conn_show_uuids, ["connection.uuid", "connection.id"])
+
+
+@parser(Specs.nmcli_conn_show_uuids)
+class NmcliConnShowUuids(CommandParser, dict):
+    """
+    This file will parse the output of nmcli connection detail information.
+
+    Sample configuration from an interface in file ``/usr/bin/nmcli conn show XXX``::
+
+        connection.id:                          System eth0
+        connection.uuid:                        5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03
+        connection.stable-id:                   --
+        connection.type:                        802-3-ethernet
+        connection.interface-name:              eth0
+        connection.autoconnect:                 yes
+
+    Examples:
+        >>> type(conn_show_uuids)
+        <class 'insights.parsers.nmcli.NmcliConnShowUuids'>
+        >>> conn_show_uuids["connection.uuid"]
+        '5fb06bd0-0bb0-7ffb-45f1-d6edd65f3e03'
+
+    Attributes:
+        data (dict): parameter-value format
+
+    """
+    def parse_content(self, content):
+        if not content:
+            raise SkipComponent()
+
+        for line in get_active_lines(content):
```

Comment:
This spec is based on a command, which won't output any `"# xxx"` comment lines, it's not recommended to use the `get_active_lines` which will iterate the lines before parsing.

Reply:
@xiangce - removed `get_active_lines`